src/backend/settings.py

```diff
@@ -407,6 +408,10 @@
 #: (see :setting:`django:TIME_ZONE` and :doc:`topics/i18n/index`)
 TIME_ZONE = "UTC"
 
+#: A string representing the time zone that is used for rendering
+#: (see :setting:`django:CURRENT_TIME_ZONE` and :doc:`topics/i18n/index`)
```

Comment:
Just for your information: The `:setting:` text role works only for official Django settings, not for our custom ones.
If you want to link to this custom setting from anywhere else, use:
```
:attr:`~backend.settings.CURRENT_TIME_ZONE`
```